tests/Stripe/Service/Issuing/CardServiceTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Stripe\Service\Issuing;
+
+class CardServiceTest extends \Stripe\TestCase
+{
+    const TEST_RESOURCE_ID = "ic_123";
+
+    private $client = null;
+    private $service = null;
+
+    /**
+     * @before
+     */
+    protected function setUpService()
+    {
+        $this->client = new \Stripe\StripeClient("sk_test_123", null, MOCK_URL);
+        $this->service = new CardService($this->client);
+    }
+
+    public function testAll()
+    {
+        $this->expectsRequest(
+            "get",
+            "/v1/issuing/cards"
+        );
+        $resources = $this->service->all();
```

Comment:
Not sure what you mean exactly, but the `$this->service->all()` invocation is only used in tests because we're manually constructing the service instance.

In actual integrations, the `StripeClient` instance will manage the service instances, so requests will look like `$stripe->issuing->cards->all()`.

Reply:
yeah I guess it's like in .net where we do service.Update() if you instantiate your own client.